src/js/components/epics/detail.tsx

```diff
@@ -343,7 +343,7 @@ const EpicDetail = (props: RouteComponentProps) => {
 
   // "Submit Epic for Review on GitHub" button:
   let submitButton: React.ReactNode = null;
-  if (readyToSubmit && project.has_push_access) {
```

Comment:
@jgerigmeyer we almost named the field the same 😂 Hope you don't mind the rename 😬